website/content/docs/release-notes/consul-terraform-sync/v0_6_0_beta.mdx

```diff
@@ -172,3 +174,82 @@ The configuration options are:
 | `agent_pool_id` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;| Optional | string | The execution mode that determines whether to use Terraform Cloud as the Terraform execution platform. Only supports "remote" or "agent". | none |
 | `agent_pool_name` | Optional | string | The name of the agent pool that should run the Terraform workloads. Only supported if `execution_mode` is set to "agent". Either `agent_pool_id` or `agent_pool_name` are required. `agent_pool_id` takes precedence over `agent_pool_name` if both are provided. | none |
 | `terraform_version` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Optional | string | The version of Terraform to use for the Terraform Cloud workspace associated with the task. Defaults to the latest compatible version supported by the organization. | none |
+
+## Health Endpoint
+The `/health` endpoint returns a successful response if Consul-Terraform-Sync (CTS) is available and running. Requests to this endpoint are not logged, which makes it suitable for health checks that constantly poll CTS.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/health` | `application/json` |
+
+### Response Statuses
+
+| Status | Reason                                               |
+| ------ | ---------------------------------------------------- |
+| 200    | CTS is healthy                                       |
+
+### Response Fields
+
+| Name         | Type   | Description                                                                        |
+| ------------ | ------ | ---------------------------------------------------------------------------------- |
+| `request_id` | string | The ID of the request. Used for auditing and debugging purposes.                   |
+
+### Example
+
+The following request makes a `GET` call to the `health` endpoint:
+
+```shell-session
+$ curl --request GET \
+  localhost:8558/v1/health
+```
+
+Response:
+
+```json
+{
+  "request_id": "b7559ab0-5111-381b-367a-0dfb7e216d41"
+}
+```
+
+## CTS Service Registration With Consul
+By default, CTS automatically registers itself as a service in Consul with a health check. The default health check is an HTTP check to the [health API endpoint](#health-endpoint). CTS deregisters itself with Consul when it is stopped.
+
+If CTS is unable to register with Consul, then it will log the error and continue without exiting.
+
+~> If ACLs are enabled in Consul, then the [Consul token](/docs/nia/configuration#token) must have the policy `service:write` for service registration to succeed.
```

Comment:
```suggestion
If ACLs are enabled in Consul, then the [Consul token](/docs/nia/configuration#token) must have the policy `service:write` for service registration to succeed.
```
I think this is just normal information, yes? No reason to render as a callout, is there?

Reply:
I called it out because CTS previously didn't have this requirement and the registration feature is enabled by default. I could add that information if that would make it more clear!

Reply:
Or in this case, does it still make sense not to make it a callout?

Reply:
I think it's fine as part of the regular content for this section in plain text. I believe that we want people to enable ACLs in a production environment, so I don't know that this would constitute exceptional or orthogonal information.

Reply:
Okay, I'll modify this to not be a callout then! I'll also add a (plain text) note specifying how this was not previously a requirement so that users may need to modify their token, though. Since these are more or less detailed release notes for the beta, I do want to make sure they understand that a token that worked for them in 0.5 may not work for this service registration feature in the 0.6.0-beta.